.golangci.yml

```diff
@@ -11,6 +11,18 @@ formatters:
   settings:
     gofumpt:
       extra-rules: true
+    goimports:
+      local-prefixes:
+        - github.com/hamba/avro
+    gci:
+      sections:
+        - standard
+        - default
+        - prefix(github.com/hamba/avro)
+        - blank
+        - dot
+        - alias
+        - localmodule
```

Comment:
I would also remove this, then in general the formatting is correct for this module.
```suggestion
```